File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    
    N = len(sizes)
    
    # 가로 < 세로로 정렬
    for i in range(N):
        sizes[i].sort()
        
    sizes = list(zip(*sizes))

    answer = max(sizes[0])*max(sizes[1])
    
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        
        int width = 0;
        int height = 0;
        
        for(int i=0; i<sizes.length; i++){
            if(sizes[i][0] > sizes[i][1]){
                if(sizes[i][0] > width){
                    width = sizes[i][0];
                }
                if(sizes[i][1] > height){
                    height = sizes[i][1];
                }
            } else {
                if(sizes[i][1] > width){
                    width = sizes[i][1];
                }
                if(sizes[i][0] > height){
                    height = sizes[i][0];
                }
            }
        }

        answer = width * height;
        
        return answer;
    }
}